client/pcr.go

```diff
@@ -106,6 +211,14 @@ type SealOpts struct {
 	Target *pb.PCRs
 }
 
+// SealOptsDirect specifies the PCR values that should be used for Seal().
+type SealOptsDirect struct {
```

Comment:
why is this struct and the UnsealOptsDirect struct in this PR? are they used somewhere?

Reply:
These structs have been removed to add to a later PR.